docs/userGuide/cliCommands.md

```diff
@@ -116,13 +119,16 @@ The caveat is that not building all pages during the initial process, or not reb
    Force live reload to process all files in the site, instead of just the relevant files. This option is useful when you are modifying a file that is not a file type monitored by the <trigger trigger="click" for="modal:cliCommands-livePreview">live preview</trigger> feature.
 
 * `-p <port>`, `--port <port>`<br>
-    Serve the website in the specified port.
+  Serve the website in the specified port.
 
+* `-v [versionNames...]`, `--versions [versionNames...]` <br>
```

Comment:
Perhaps can mention that this will override the attribute in `site.json`? Same for the `build` command